lib/api/src/modules/shortcuts.ts

```diff
@@ -118,142 +118,144 @@ export default function initShortcuts({ store }: Module) {
     handleShortcutFeature(fullApi, feature) {
       const {
         layout: { isFullscreen, showNav, showPanel },
+        ui: { enableShortcuts },
       } = store.getState();
-
-      switch (feature) {
-        case 'escape': {
-          if (isFullscreen) {
-            fullApi.toggleFullscreen();
-          } else if (!showNav) {
-            fullApi.toggleNav();
+      if (enableShortcuts) {
```

Comment:
Can we just exit early here if `!enableShorcuts`?

Reply:
Changed it.